sdv/datasets/local.py

```diff
@@ -6,12 +6,15 @@
 from sdv.utils import load_data_from_csv
 
 
-def load_csvs(folder_name):
+def load_csvs(folder_name, pandas_kwargs=None):
     """Load csv files from specified folder.
 
     Args:
         folder_name (str):
             The full path of the folder with the data to be loaded.
+        pandas_kwargs (dict):
```

Comment:
@npatki Should we use this name or the one in the issue? The reason I ask is that the `SingleTableMetadata.detect_from_csv` function calls it `pandas_kwargs` so maybe we should be consistent

Reply:
There is currently no parameter listed in the API docs for `detect_from_csv(s)` for either [Single Table](https://docs.sdv.dev/sdv/single-table-data/data-preparation/single-table-metadata-api#auto-detect-metadata) or [Multi Table](https://docs.sdv.dev/sdv/multi-table-data/data-preparation/multi-table-metadata-api#creation-api) data so we can choose to call it what we want

Back when we had a singular `load_csv` method, we had  called the parameter `read_csv_parameters` so that's why it was carried over into the issue requirements.

So I'm onboard with the reverse: Let's name the parameter `read_csv_parameters` for the single/multi table metadata `detect_from_csv(s)` method (align it to what we describe here in this issue). We could make this a separate issue?

Reply:
Changed param name to `read_csv_parameters`